### PR TITLE
Add rb-fluentd

### PR DESCRIPTION
**Pull request by oblute**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by oblute** [comment]
@benhuff agree to be a maintainer?

**Comment by benhuff** [comment]
yes 🎉

**Comment by oblute** [comment]
@conda-forge/help-ruby this recipe is ready for review

File: recipes/rb-fluentd/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    - gem unpack {{ name }}-{{ version }}.gem
+  skip: {{ win }}
+
+requirements:
+  host:
+    - ruby
+  run:
+    - ruby
+    - rb-cool.io >=1.4.5,<2.0.0
+    - rb-dig_rb >=1.0,<2
```
Comment (sodre) [inline recipes/rb-fluentd/meta.yaml]:
```suggestion
    - rb-dig_rb >=1.0.0,<1.1
```

File: recipes/rb-fluentd/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    - rb-http_parser.rb >=0.5.1,<0.7.0
+    - rb-msgpack >=1.2.0,<2.0.0
+    - rb-serverengine >=2.0.4,<3.0.0
+    - rb-sigdump >=0.2.2,<1.0.0
+    - rb-strptime >=0.2.2,<1.0.0
```
Comment (sodre) [inline recipes/rb-fluentd/meta.yaml]:
```suggestion
    - rb-strptime >=0.2.2,<0.3
```

File: recipes/rb-fluentd/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+source:
+  url: https://rubygems.org/downloads/{{ name }}-{{ version }}.gem
+  sha256: 20c90d24d27051c6d27c18d34540a23aa802a375e3a61d69b338faba8feb8934
+
+build:
+  noarch: generic
+  number: 0
+  script:
+    - gem install -N -l -V --norc --ignore-dependencies {{ name }}-{{ version }}.gem
+    - gem unpack {{ name }}-{{ version }}.gem
```
Comment (sodre) [inline recipes/rb-fluentd/meta.yaml]:
```suggestion
      && gem unpack {{ name }}-{{ version }}.gem
```

**Pull request merged**